app/views/books/show.html.erb

```diff
@@ -17,3 +19,25 @@
    <%= @book.user.first_name %>
   </h5>
 </p>
+
+Reviews for <%= @book.title %>:
+<ul>
+<% @reviews.each do |review|%>
+  <li>
+    <h3><%= review.title %></h3> Votes: <%= review.votes %><br>
+    <%= review.user.first_name.chars.first.capitalize %> <%= review.user.last_name %>
```

Comment:
put this logic in your controller (or better yet, a method on your Review model!!)